WatsonTcp/WatsonTcpClient.cs

```diff
@@ -375,6 +375,8 @@ public void Start()
                 Logger?.Invoke("[WatsonTcpClient] Connecting to " + _ServerIp + ":" + _ServerPort);
 
                 _Client.LingerState = new LingerOption(true, 0);
+                byte[] outValue = new byte[10];
+                _Client.Client.IOControl(IOControlCode.KeepAliveValues, new KeepAliveValues().Values, outValue);
```

Comment:
This is windows specific and throws an exception on all other systems.

Reply:
```cs
socket.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.KeepAlive, true);
socket.SetSocketOption(SocketOptionLevel.Tcp, SocketOptionName.TcpKeepAliveTime, 5); // 5 seconds
socket.SetSocketOption(SocketOptionLevel.Tcp, SocketOptionName.TcpKeepAliveInterval, 5); // 5 seconds
socket.SetSocketOption(SocketOptionLevel.Tcp, SocketOptionName.TcpKeepAliveRetryCount, 1); 
```
This is supposed to do the same and works for me (linux 5.8, dotnet 3.1)
Does this also work on windows?